backend/dataall/api/Objects/DatasetTable/resolvers.py

```diff
@@ -12,7 +12,7 @@
 from ....db import permissions, models
 from ....db.api import ResourcePolicy, Glossary
 from ....searchproxy import indexers
-from ....utils import json_utils
+from ....utils import json_utils, sql_utils
```

Comment:
I see that this feature branch has been created from main instead than from v1.6 branch. maybe we get some issues when merging back to main